docs/book/intro.md

```diff
@@ -5,3 +5,132 @@ laminas-cli is a command-line interface for Laminas projects.
 It provides seamless integration with Laminas MVC and Mezzio applications.
 
 It is easily extensible by custom command. 
+
+## Integration with component
+
+If you'd like to add integration with laminas-cli into your components
+you have to the following:
+
+1. Use `laminas/laminas-cli` as dev dependency:
```

Comment:
Why a `dev` dependency? I often have CLI scripts I want to use in production as well, and it seems this is allowing the ability to add arbitrary commands to the tooling?

Reply:
This is instruction for integration with other libraries, not with the application.
In the application it should be installed as non-dev dependency if user needs to run any commands on production.

For other libraries - let say mezzio we'd like to install it as dev dependency and add the commands there. It should be suggested package tho, so if user want to use cli commands should install laminas-cli in the project.

We will add this dependency to require in both skeletons: MVC and Mezzio.